paper/paper.md

```diff
@@ -0,0 +1,88 @@
+<!--
+SPDX-FileCopyrightText: ASSUME Developers
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+---
+title: 'ASSUME - Agent-based Simulation for Studying and Understanding Market Evolution'
```

Comment:
The SoftwareX which is more functionality and research focused paper is named "ASSUME: Agent-based Simulation for Studying and Understanding Evolution of Electricity Markets". Here I would the same title as our github for better recognition, namely "ASSUME: Agent-Based Electricity Markets Simulation Toolbox"

Reply:
Good point.
I always thought that ASSUME is an acronym for exactly this, I don't know why our README says "ASSUME: Agent-Based Electricity Markets Simulation Toolbox".
So I think that it makes sense to unify this..?

Reply:
there is really no english acronym for assume, we made it up. the original name comes from 
**A**gentenbasierte **S**trommarkt**S**imulation zur **U**ntersuchung von **M**arktdesigns für das zukünftige **E**nergiesystem